css/style.css

```diff
@@ -0,0 +1,169 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@300;400;500;600&display=swap');
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  font-family: "Poppins", sans-serif;
+}
+
+header {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  align-content: center;
+  padding: 15px;
+  max-width: 375px;
+  margin: 0 auto;
+}
+
+nav {
+  display: flex;
+  align-items: center;
+  height: 50px;
+  width: 100%;
+}
+
+ul {
+  list-style: none;
+}
+
+.logo-part {
+  flex: 1;
+  padding: 20px;
+  font-weight: 800;
+  font-size: 24px;
+}
+
+.logo-text {
+  color: #6070ff;
+}
+
+.menu-part {
+  display: flex;
+  gap: 50px;
+  justify-content: center;
+  align-items: center;
+  flex-wrap: wrap;
+  text-decoration: none;
+  list-style: none;
+}
+
+.menu-part a {
+  text-decoration: none;
+  font-size: 25px;
+  line-height: 20px;
+  color: #344563;
+  font-weight: 500;
+  padding: 10px;
+  align-items: center;
+  font-style: bold;
+}
+
+.hamburgur {
+  padding: 10px;
+}
+
+.bar {
+  width: 30px;
+  height: 3px;
+  margin: 4px;
+  background: #6070ff;
+  display: flex;
+}
+
+section {
+  display: block;
+  margin: 0 auto;
+  max-width: 375px;
+}
+
+.background {
+  background: url(/images/header-shapes-mobile.svg);
```

Comment:
- [ ] I think your background image is not being loaded properly as it is returning `404 Not Found`. It seems there is an error in url path. Kindly fix this to match it with the Figma template  👍🏽 🙂 
Your design 👇🏽 
<img width="335" alt="Screenshot 2022-12-01 at 7 00 00 PM" src="https://user-images.githubusercontent.com/24832846/205065253-fca3eb99-9859-4cda-87cc-be9deb91abe5.png">